src/main/java/razdwatrzy/zzpj/model/UserCredentials.java

```diff
@@ -0,0 +1,37 @@
+package razdwatrzy.zzpj.model;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "user_credentials")
+public class UserCredentials {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @OneToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
+    @NotNull
+    @Email
+    @Column(unique = true)
+    private String email;
+
+    @NotNull
+    @Column
+    private String password;
```

Comment:
Is there a 'password" type in database, it'll be more secure.

Reply:
I did not find anything like that